src/backend/joanie/core/utils/synchronize.py

```diff
@@ -0,0 +1,71 @@
+"""
+Utils that can be useful throughout Joanie's core app to synchronize data with other services
+"""
+import hashlib
+import hmac
+import json
+import logging
+
+from django.conf import settings
+
+import requests
+from urllib3.util import Retry
+
+logger = logging.getLogger(__name__)
+
+adapter = requests.adapters.HTTPAdapter(
+    max_retries=Retry(
+        total=4,
+        backoff_factor=0.1,
+        status_forcelist=[500],
+        allowed_methods=["POST"],
+    )
+)
+
+session = requests.Session()
+session.mount("http://", adapter)
+session.mount("https://", adapter)
+
+
+class SynchronizeHook:
+    """Class to synchronize data with other services"""
+
+    @staticmethod
+    def submit_hook(course_runs):
```

Comment:
Since it is just a static method it should just be a function?

Maybe call it `call_webhook_to_synchronize_course_runs`?

Reply:
One of a consequence to implement it in a function instead of a static, which I discover doing it, is that you specifically need to mock the function from where you call it. Doing this implies changing the test and splitting the mock between the call made from the Product model and the CourseRun model. IMO one of the points of testing the synchronization is to make sure we don't have extra calls that submit hooks from another signal that we wouldn't have thought about. It wasn't calculated this way but to have more secure tests I would actually leave the static. WDYT?